schemas/test.js

```
@@ -0,0 +1,37 @@
+import $RefParser from "@apidevtools/json-schema-ref-parser";
+import Ajv2019 from "ajv/dist/2019.js"
+import fs from 'fs';
+import yaml from 'js-yaml';
+
+function stripIds(schema, first) {
+	if (schema !== null && typeof schema === 'object') {
+		if (!first) {
+			// Every referenced schema we pull in should have its $id and $schema stripped, or ajv complains
+			// Skip the root object, as that should retain the $schema and $id
+			delete schema.$id;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)